src/Test.UnitTests.Sarif/Core/PropertyBagHolderTests.cs

```diff
@@ -4,11 +4,12 @@
 using System;
 using System.Collections.Generic;
 using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.Sarif
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Core
```

Comment:
>Test [](start = 33, length = 4)

We're standardizing the test namespaces to what VS generates by default when you create a test class. As matters stand, our unit tests are in a hodgepodge of namespaces, which makes them difficult to find in the Test Explorer.